src/beautifiers/black.coffee

```diff
@@ -37,3 +47,15 @@ module.exports = class Black extends Beautifier
       onStdin: (stdin) ->
         stdin.end text
     })
+    .then((stdout) =>
+        console.log stdout
+        if options.sort_imports
+            filePath = context.filePath
+            projectPath = atom?.project.relativizePath(filePath)[0]
+            @exe("isort").run(["-sp", projectPath, "-"], {
```

Comment:
- [x] Confirm `-` enables stdin for `isort`: https://github.com/timothycrosley/isort/blob/90fb6eef6e694fd0316917e3c481390758cd5efc/isort/main.py#L335-L336